src/main/kotlin/com/reicheltp/celtic_rituals/init/ModEntities.kt

```diff
@@ -0,0 +1,18 @@
+package com.reicheltp.celtic_rituals.init
+
+import com.reicheltp.celtic_rituals.rituals.bag.RitualBagEntity
+import net.minecraft.entity.EntityClassification
+import net.minecraft.entity.EntityType
+import net.minecraft.tileentity.TileEntityType
+import net.minecraftforge.registries.ObjectHolder
+
+object ModEntities {
+    @ObjectHolder("celtic_rituals:ritual_bag_entity")
+    var RITUAL_BAG_ENTITY: EntityType<RitualBagEntity> =
```

Comment:
Use ObjectHolder instead of creating the entity. This allows other to override this.